pkg/console/operator/sync_v400.go

```diff
@@ -64,6 +63,7 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	rt, rtChanged, rtErrReason, rtErr := co.SyncRoute(set.Operator)
 	toUpdate = toUpdate || rtChanged
 	status.HandleProgressingOrDegraded(updatedOperatorConfig, "RouteSync", rtErrReason, rtErr)
+	co.CheckRouteHealth(updatedOperatorConfig, rt)
```

Comment:
We can check route health right away, rather than waiting for the whole loop to complete. This should let us aggregate status.